Jenkinsfile

```diff
@@ -69,6 +69,8 @@ pipeline {
                         dir('src/github.com/docker/app') {
                             checkout scm
                             sh 'make -f docker.Makefile coverage'
+                            archiveArtifacts '_build/ci-cov/all.out'
+                            archiveArtifacts '_build/ci-cov/coverage.html'
```

Comment:
Why do we need them now and not before?

Reply:
@mat007 this is _sadly_ what triggers a build fail or not (i.e. if the artifacts are here or not)